app/api/helpers/auth.py

```diff
@@ -31,3 +31,14 @@ def is_verified_user():
     @staticmethod
     def is_accessible():
         return current_user.is_authenticated
+
+    @staticmethod
+    def check_auth_admin(username, password):
+        # This function is called to check for proper authentication & admin rights
+        if username and password:
+            user = User.query.filter_by(_email=username).first()
+            if user:
+                if user.is_correct_password(password):
+                    if user.is_admin:
```

Comment:
Hi @mrsaicharan1, I think its better that lines 40, 41, 41 be reduced to only 1

Reply:
@mrsaicharan1 yes please use `and` and combine the logics